statics/js/components/preference.js

```diff
@@ -0,0 +1,98 @@
+/* jshint multistr: true */
+
+var PreferenceComponent = {
+
+  name: 'Preference',
+
+  mixins: [notificationMixin],
+
+  template: '\
+    <form class="form-preference" @submit.prevent="save">\
+      <h2> Preferences </h2>\
+      <div class="form-group">\
+        <label for="collapse-index">Collapse Index</label>\
+        <input id="collapse-index" type="number" class="form-control input-sm" v-model="collapseIndex" min="0" />\
+      </div>\
+      <div class="form-group">\
+        <div class="row">\
+          <label for="default-filter">Filters</label>\
+          <button class="btn btn-primary btn-round-xs btn-xs" type="button" @click="addFilter">+</button>\
```

Comment:
I would prefer to have the "+" button below the list of fields

Reply:
moved